toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -56,6 +56,11 @@ func doCustomizations(baseConfigPath string, config *imagecustomizerapi.SystemCo
 		return err
 	}
 
+	err = enableOrDisableServices(config.Services, baseConfigPath, imageChroot)
```

Comment:
I think this should sit between the users step and the `PostInstallScript` step.